free-2.md

```diff
@@ -11,41 +11,118 @@ _What need is met by your idea? Who are you helping? What is that person's
 
+A DSL would allow for more customization in order to effectively
+filter out emails. 
+This customization is key since different users would want to receive and 
+filter out different types of emails.
+The customization could also be used to specifically target types of emails.
+For example, a user could specify that they do not want emails that only contain
+numbers and a sign-off. 
+This would then filter out counting threads.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I don't mind nonsensical emails from dorm chats but a lot of people do.
+I would be happy to help people eliminate annoyances in their lives.
+
+I came up with the idea when someone mentioned that another person has a really
+complex email filtration system.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Filter out unwanted email types.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would write a configuration file. This file would contain something
+like
+
+```
+name: "ISO war"
+keywords: ["ISO"]
+start-frequency: 60
+start-count: 3
+step-frequency: 3600
+to: "ISO"
+```
+
+This means, filter out emails that are considered to be "ISO war" emails. These
+emails contain the word "ISO". 
+Filter them out if more than 3 such emails are 
+sent over the course of 60 seconds and continue filtering them out until the 
+frequency drops below 1 email per 3600 seconds. 
+These filtered emails should go to a folder labeled "ISO".
+
+A configuration file using JSON or XML-like syntax would give the system a good
+level of customization, provided that the language has enough keys.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The user would run a command, maybe in the command line, to start the program.
+The program reads the configuration file and continuously reads incoming emails.
+It would filter emails that should be filtered at all times and it would
+filter emails once certain criteria (such as emails per second) are 
+triggered.
+
+Syntax errors could occur if users try to use a key that the program does not
+recognize. This error should be displayed after the user enters the command. 
+
+There could be a lot of silent errors where the program misinterprets the user's
+input and simply fails to filter out the specified emails or filters out emails
+the user wants to receive. It would be on the user to identify these errors.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to identify types of emails based on keywords and regular
+expressions.
+
```

Comment:
regular expressions :) very nice. I think if possible the first two claims should both be easy but I understand that may be tricky. I agree no loops!